apps/mail/components/ui/employee-hover-card.tsx

```
@@ -0,0 +1,121 @@
+              </div>
+            </div>
+
+            {/* Location */}
+            <div className="flex items-center gap-2 text-white/60">
+              <svg 
+                width="16" 
+                height="16" 
+                viewBox="0 0 24 24" 
+                fill="currentColor"
+                className="w-4 h-4 opacity-60"
+              >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)